configure.ac

```diff
@@ -4,7 +4,7 @@ AC_PREREQ([2.5])
 ##########################################################################
 m4_define([odpapi_generation_version], [1])
 m4_define([odpapi_major_version], [21])
-m4_define([odpapi_minor_version], [0])
+m4_define([odpapi_minor_version], [1])
```

Comment:
Is this PR intended to be its own release? If yes, then CHANGELOG updates should also be included in this PR.

Reply:
Not a release (no tag), but API development in master. Minor version number increment highlights that API is not any more at release 1.21 level, although it's backwards compatible to that. Next release will be tagged as v1.22.